Sources/DeepLearning/Layer.swift

```diff
@@ -1295,3 +1295,59 @@ public struct Reshape<Scalar: TensorFlowFloatingPoint>: Layer {
         return input.reshaped(toShape: shape)
     }
 }
+
+struct LSTMCell<Scalar: TensorFlowFloatingPoint>: Layer {
```

Comment:
Make this and all properties public, otherwise people can’t use it :)

Reply:
Done.